cms/templates/studio_xblock_wrapper.html

```diff
@@ -128,14 +132,16 @@
     <div class="wrapper-xblock-message xblock-validation-messages" data-locator="${xblock.location | h}"/>
 % endif
 
-% if is_root or not xblock_url:
-    <article class="xblock-render">
-    ${content}
-    </article>
-% else:
-    <div class="xblock-message-area">
-    ${content}
-% endif
+% if show_preview:
+    % if is_root or not xblock_url:
+        <article class="xblock-render">
+        ${content}
+        </article>
+    % else:
+        <div class="xblock-message-area"></div>
```

Comment:
Should the close div be AFTER the content? I think that is the behavior I am seeing in prod right now.